spec/controllers/sessions_controller_spec.rb

```diff
@@ -22,6 +22,16 @@
       it 'redirects to the home page' do
         expect(get(:index)).to redirect_to summaries_url
       end
+
+      context 'when user requested a page before authentication' do
```

Comment:
I tried a fresh browser request of localhost/holds and it redirected to the homepage after authentication. Is that expected behavior? It seems, if I'm reading the logic correctly that unless my request is for the root it should send me to my requested for destination. Right?

Reply:
right, It should be redirecting to `/holds`. you are reading correctly, not sure why it did not work.

Reply:
we good